net/sslscan/Portfile

```diff
@@ -2,9 +2,13 @@
 
 PortSystem      1.0
 
-PortGroup       github      1.0
-PortGroup       makefile    1.0
-PortGroup       openssl     1.0
+PortGroup       github          1.0
+PortGroup       legacysupport   1.1
+PortGroup       makefile        1.0
+PortGroup       openssl         1.0
+
```

Comment:
@barracuda156 I recommend not to make unneeded whitespace changes to ports you don't maintain

Reply:
@reneeotten I would look super-ugly, if one PG is aligned differently. Since here 1.0 were aligned to multiples of 4, I shifted them to the next multiple.

Reply:
no, that's how it is done in almost every other Portfile

Reply:
@reneeotten I will not argue here, but I wanna make sure we are clear on this. In most of portfile 1.0 are NOT aligned to multiples of four, but simply follow PG name, so no problem arises.

Compare any normal portfile:
```
PortGroup           compiler_blacklist_versions 1.0
PortGroup           compilers 1.0
PortGroup           github 1.0
PortGroup           linear_algebra 1.0
PortGroup           legacysupport 1.1
```
vs what we have here:
```
PortGroup       github      1.0
PortGroup       makefile    1.0
PortGroup       openssl     1.0
```

Reply:
the last option is fine, there is no need to have the version aligned with multiples of four; anyway I'll leave it to @raimue as the maintainer of the port

Reply:
If everyone is fine with the last option, I am okay with that. I just wanted to make it look consistent, but it is non-functional, so not important.

Reply:
I personally prefer when everything, including versions are aligned, but that's just my personal taste.
```
PortGroup       github          1.0
PortGroup       makefile        1.0
PortGroup       openssl         1.0
PortGroup       legacysupport   1.1
```

Reply:
@herbygillot Well, that is what I did, following the style of the portfile in question.

I am indifferent re aligning versions, but think it should be either this or that, not a random mixture of the two LOL

Reply:
I have no strong preference whether port group name and version should be aligned in columns. I probably maintain ports that use both styles. Without actual fact checking, my feeling is that having port group name and version separated by a single space is more common.

I'll add another commit that changes the style to avoid such changes in the future.